torch/csrc/autograd/python_function.cpp

```diff
@@ -810,6 +813,7 @@ PyObject *THPFunction_apply(PyObject *cls, PyObject *inputs)
   InputFlags& input_info = info_pair.second;
 
   // Initialize backward function (and ctx)
+  bool is_executable = input_info.flags.is_executable;
```

Comment:
it's a bit weird that we keep `is_executable` in this file, because it has no meaning anymore. It would be nice to replace it with sth else e.g. "any input requires grad"

Reply:
I'll clean this up in the PR that removes volatile (should be ready soon).